lib/slack/real_time/concurrency/async.rb

```diff
@@ -15,7 +15,22 @@ class Socket < Slack::RealTime::Socket
           def start_async(client)
             Thread.new do
               ::Async::Reactor.run do
```

Comment:
You can simplify this code by something like 

```ruby
::Async::Reactor.run do |task|
  task.async do
    # nested task
  end
end
```

Reply:
Great point! I'll fix this.